spec/requests/api/service_dialogs_spec.rb

```diff
@@ -87,6 +87,14 @@
       expect_query_result(:service_dialogs, dialogs.count, dialogs.count)
       expect_result_resources_to_include_data("resources", "label" => dialogs.pluck(:label))
     end
+
+    it "query service dialogs of a Template fetches content via the virtual attribute accessor" do
```

Comment:
np, will update.  We do need to go through each of the send's in the base_controller to make sure all target methods are still accessible with the new API controller namespaces when you get a chance.  Just to make sure there are no other surprises.  Thanks.